vcpkg.json

```diff
@@ -158,6 +170,10 @@
       "name": "trantor",
       "version": "1.5.11#2"
     },
+    { 
+      "name": "ffmpeg",
+      "version": "4.4"
```

Comment:
Could you remind me what error you got when you moved this version override into rocal's vcpkg.json?

Reply:
Below is when I put the override into rocal's manifest file.
```
error: version conflict on ffmpeg:x64-linux-dynamic: baseline required 5.1.2#6 but vcpkg could not compare it to 4.4.
The two versions used incomparable schemes:
    "4.4" was of scheme string
    "5.1.2#6" was of scheme relaxed
This can be resolved by adding an explicit override to the preferred version, for example:
    "overrides": [
        { "name": "ffmpeg", "version": "4.4" }
    ]
```

Vcpkg basically requires the override to be in the main manifest file.

Reply:
Thank you. Just wanted it for future reference